ssh-tor add note (#642)
Start9Labs/documentation: site/source/misc-guides/ssh-tor.rst

> ======================

Reviewer: ```suggestion
==================
```

Using SSH Over Tor

> ======================

Reviewer: ```suggestion
==================
```

> .. note:: The following guide requires that you have already added an :ref:`SSH key to your server<ssh>` and run :ref:`Tor s a global process on your client<connecting-tor>`.

Reviewer: ```suggestion
.. note:: The following guide requires that you have already added an :ref:`SSH key to your server<ssh>` and run :ref:`Tor as a background process on your client<connecting-tor>`.
```
